src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -207,6 +207,10 @@ public function testAddContent()
         $crawler->addContent('<html><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><span>中文</span></html>');
         $this->assertEquals('中文', $crawler->filterXPath('//span')->text(), '->addContent() guess wrong charset');
 
+        if (!extension_loaded('iconv')) {
+            $this->markTestSkipped('Extension iconv is required.');
+        }
+
         $crawler = new Crawler();
         $crawler->addContent(iconv('UTF-8', 'SJIS', '<html><head><meta charset="Shift_JIS"></head><body>日本語</body></html>'));
```

Comment:
this test should be moved to a separate test, to skip only this instead of marking the whole ``testAddContent`` as skipped